modules/nodejs/manifests/repo.pp

```diff
@@ -0,0 +1,20 @@
+# == Class: nodejs::repo
+#
+# Use our own mirror of the nodejs repo. Should be used with `manage_repo`
+# disable of the upstream module.
+#
+# === Parameters
+#
+# [*apt_mirror_hostname*]
+#   Hostname to use for the APT mirror.
+#
+class nodejs::repo(
+  $apt_mirror_hostname = undef,
+) {
+  apt::source { 'nodejs':
+    location     => "http://${apt_mirror_hostname}/nodejs",
+    release      => $::lsbdistcodename,
+    architecture => $::architecture,
+    key          => '',
```

Comment:
The key should be the same as all the others. Check across the codebase for where we have used `apt::source` previously.